di-ipv-credential-issuer-stub/src/test/java/uk/gov/di/ipv/stub/cred/handlers/AuthorizeHandlerTest.java

```diff
@@ -261,6 +262,37 @@ void doAuthorizeShouldRenderMustacheTemplateWhenValidRequestReceivedWithEncrypte
                                 .toJSONObject()
                                 .get(SHARED_CLAIMS);
         assertEquals(gson.toJson(claims), frontendParamsCaptor.getValue().get("shared_claims"));
+        assertFalse((Boolean) frontendParamsCaptor.getValue().get(CRI_MITIGATION_ENABLED_PARAM));
+    }
+
+    @Test
+    void doAuthorizeShouldRenderMustacheTemplateWhenValidRequestReceivedWithMitigationEnabled()
```

Comment:
I think, need to add new test methods to cover the code changes mentioned above, don't you think?

Reply:
done